code/modules/research/designs/anomaly.dm

```diff
@@ -0,0 +1,29 @@
+/datum/design/scienceglasses
+	name = "Science Goggles"
+	desc = "You expect those glasses to protect you from science-related hazards. Maybe you shouldn't."
+	id = "scienceglasses"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 700, MAT_GLASS = 2000)
+	category = "Anomaly"
+	build_path = /obj/item/clothing/glasses/science
+
+/datum/design/depth_scanner
+	name = "Depth Analysis Scanner"
+	desc = "Used to check spatial depth and density of rock outcroppings."
+	id = "depth_scanner"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 500, MAT_GLASS = 1000, MAT_GOLD = 200, MAT_SILVER = 200)
+	category = "Anomaly"
+	build_path = /obj/item/device/depth_scanner
+
+/datum/design/ano_scanner
```

Comment:
>This is completely against SS13's core concepts.

Implying that jumping into space hoping to find some fun, or loitering on the asteroid only to come back on the station and wonder what happenned isn't part of SS13's core concepts. There's enough room on the server for both the kind who wants to interact and the one that doesn't.

What I do agree though is that those machines are stupid, and having high RP stuff in the workplace of those who are the less likely to RP (outside their own heads) is even more stupid.